lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -21,41 +21,65 @@ void main() {
   internalBootstrapBrowserTest(() => testMain);
 }
 
+late EngineFlutterView view;
+DomElement get rootElement => view.dom.rootElement;
+
 void testMain() {
-  ensureFlutterViewEmbedderInitialized();
-  final DomElement rootElement =
-      EnginePlatformDispatcher.instance.implicitView!.dom.rootElement;
-  late double dpi;
+  final DomElement hostElement = createDomHTMLDivElement();
 
-  setUp(() {
-    ui.PlatformDispatcher.instance.onPointerDataPacket = null;
-    dpi = EngineFlutterDisplay.instance.devicePixelRatio;
+  setUpAll(() {
+    domDocument.body!.append(hostElement);
+    // Remove <body> margins to avoid messing up with all the test coordinates.
+    domDocument.body!.style.margin = '0';
```

Comment:
I guess we didn't need this before because the (full-screen) implicitView has [this](https://github.com/flutter/engine/blob/main/lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/full_page_embedding_strategy.dart#L67)?